lib/service_check.pm

```diff
@@ -228,11 +228,6 @@ service package name.
 
 sub _is_applicable {
     my ($srv_pkg_name) = @_;
-    if ($srv_pkg_name eq 'kdump' && is_s390x) {
-        # workaround for bsc#116300 on s390x
-        record_soft_failure 'bsc#1163000 - System does not come back after crash on s390x';
-        return 0;
-    }
```

Comment:
don't we need to convert it into record_info, that `record 0;` is an early exit that probably is needed.

Reply:
We would if the issue reported was still reproducible, which is not the case.